assets/src/edit-story/components/panels/test/textStyle.js

```diff
@@ -45,7 +45,7 @@ const DEFAULT_PADDING = { horizontal: 0, vertical: 0, locked: true };
 
 function Wrapper({ children }) {
   return (
-    <FlagsProvider features={{ newFontPicker: false }}>
+    <FlagsProvider>
```

Comment:
Worth keeping that when it's not used any longer?

Reply:
Not really, thanks!